src/js/components/common/expandable-attribute.js

```diff
@@ -57,7 +57,7 @@ export const ExpandableAttribute = ({
       copy(secondary);
       setSnackbar('Value copied to clipboard');
     }
-    if (!expanded) {
+    if (!expanded && !!onExpansion) {
```

Comment:
ok, this is what we want, right, if onExpansion===true, just to double check.

Reply:
yes, we call it only if it is defined